bot.js

```diff
@@ -70,6 +70,8 @@ bot.on('message', function(message) {
     var monkey_island_match = monkey_island.search(message.text);
     if (monkey_island_match) {
       replyText(monkey_island_match, message);
+    } else if (message.text.search(/mue?tt?(er|i)/i) != -1) {
```

Comment:
Please use `/mue?tt?(er|i)/i.test(message.text)` as on line 75 for more consistent code.